routes/index.js

```diff
@@ -31,7 +31,7 @@ var routes = {
   },
   'stripe': function(request, reply) {
     var transaction = request.payload || {};
-    if (transaction.recurring === 0) {
+    if (transaction.frequency === 'single') {
```

Comment:
Having a 0/1 values stored was hard to read. We also wanted the query string to be readable too. So I figure just always keep the same values here.